eternalcore-api/src/main/java/com/eternalcode/core/feature/warp/event/PreWarpTeleportEvent.java

```diff
@@ -0,0 +1,51 @@
+package com.eternalcode.core.feature.warp.event;
+
+import com.eternalcode.core.feature.warp.Warp;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+ * Called before teleportation to warp.
```

Comment:
```suggestion
/**
 * Called before teleportation to warp.
 */
```

Reply:
My bad, suggested only one line, changed 3 instead.

(first try of doing review in intelliJ as suggested by Rollczi, to better see changes)